src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -430,9 +430,9 @@ export const FIELDS: Readonly<Record<FieldKey, ColumnType>> = {
   [FieldKey.STACK_MODULE]: 'string',
   [FieldKey.STACK_FUNCTION]: 'string',
   [FieldKey.STACK_IN_APP]: 'boolean',
-  [FieldKey.STACK_COLNO]: 'number',
-  [FieldKey.STACK_LINENO]: 'number',
-  [FieldKey.STACK_STACK_LEVEL]: 'number',
+  [FieldKey.STACK_COLNO]: 'string',
```

Comment:
We do store this as a number, but I can see how not having it show up in the autocomplete for percentile() is helpful.